src/main/webapp/dev/js/main/feedbackSubmissionsEdit.es6

```diff
@@ -627,6 +627,64 @@ function updateConstSumMessages() {
     }
 }
 
+function getMaxSelectableMsqChoices(qNum) {
+    const $input = $(`input[name="msqMaxSelectableChoices-${qNum}"]`);
+
+    return $input.prop('disabled') ? Number.MAX_SAFE_INTEGER : $input.val();
+}
+
+function getMinSelectableMsqChoices(qNum) {
+    const $input = $(`input[name="msqMinSelectableChoices-${qNum}"]`);
+
+    return $input.prop('disabled') ? 0 : $input.val();
+}
+
+function validateMsqQuestions() {
+    const msqQuestionNums = getQuestionTypeNumbers('MSQ');
+
+    // validate min/max selectable choices restrictions
+    for (let i = 0; i < msqQuestionNums.length; i += 1) {
```

Comment:
Consider instead:
```javascript
for (let i in msqQuestionNums) {
```

Reply:
Current linter does not allow `for in` statements.